cron/data/iterator_test.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Vershandlern 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permisshandlerns and
+// limitathandlerns under the License.
+
+package data
+
+import (
+	"os"
+	"testing"
+
+	"github.com/ossf/scorecard/repos"
+)
+
+type outcome struct {
+	repo     repos.RepoURL
+	hasError bool
+}
+
+func TestCsvReader(t *testing.T) {
+	t.Parallel()
+	testcases := []struct {
+		name     string
+		filename string
+		outcomes []outcome
+	}{
+		{
+			name:     "Basic",
+			filename: "testdata/basic.csv",
+			outcomes: []outcome{
+				{
+					hasError: false,
+					repo: repos.RepoURL{
+						Host:  "github.com",
+						Owner: "18F",
```

Comment:
nit: should just use fake repo names instead of these weird repo names 18F/1N3, etc

Reply:
Done.